openstack/ironic/templates/redfish-certrobot-cronjob.yaml

```diff
@@ -43,6 +45,26 @@ spec:
             - -m
             - redfish_certrobot
             env:
+            - name: BMC_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}-redfish-certrobot
+                  key: ipmi_user_name
+            - name: BMC_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}-redfish-certrobot
+                  key: ipmi_user_passwd
+            - name: VAULT_REGION
+              value: "{{ .Values.vault.region }}"
+            - name: VAULT_ADDR
+              value: "{{ .Values.vault.addr }}"
+            - name: VAULT_PROJECT
+              value: "{{ .Values.vault.project }}"
+            - name: NETBOX_URL
+              value: "{{ .Values.netbox.url }}"
+            - name: VAULT_ENV
```

Comment:
We do not need any vault variables anymore. That should be covered by the secrets-injector.